packages/cubejs-clickhouse-driver/docker-compose.yml

```diff
@@ -0,0 +1,10 @@
+version: '2'
```

Comment:
Let's set version to `3`, other than that the format of the file should be the same in our case.